.eslintrc.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
         // Disabled because some packages contain subcomponents. Our documentation system currently
         // requires that they be defined in one file.
         'react/no-multi-comp': 'off',
+
+        // React hooks
+        'react-hooks/rules-of-hooks': 'error',
+        'react-hooks/exhaustive-deps': 'warn',
```

Comment:
```suggestion
        'react-hooks/exhaustive-deps': 'error',
```

Let's do error for both? I'd rather fail the build and add `eslint-disable` lines (with comments) rather than having warnings in the logs that are easy to miss and don't include context.